### PR TITLE
chore(repo): rename branch master to main

### PR DESCRIPTION
**Pull request by kmova**
- As part of inclusive naming, work towards renaming branches
  with name master to main or develop
- Updated the referece to base apline to latest

Signed-off-by: kmova <kiran.mova@mayadata.io>

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ on:
     branches:
       # on pull requests to master and release branches
```
Comment (akhilerm) [inline .github/workflows/pull_request.yml:20]:
```suggestion
      # on pull requests to main and release branches
```

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -19,6 +19,7 @@ on:
...
       - master
```
Comment (akhilerm) [inline .github/workflows/pull_request.yml:21]: Do we need to still have the master here?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -32,7 +32,7 @@ jobs:
         run: |
           BRANCH="${GITHUB_REF##*/}"
           CI_TAG=${BRANCH#v}-ci
-          if [ ${BRANCH} = "master" ]; then
+          if [ ${BRANCH} = "main" ]; then
```
Comment (akhilerm) [inline .github/workflows/build.yml:35]: There is reference [here](https://github.com/openebs/linux-utils/blob/24b0576aa8779594e5eb542cbed7b858f427d5fd/.github/workflows/build.yml#L117) to master. Please change that also.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -13,26 +13,26 @@ OpenEBS is an Apache 2.0 Licensed project and all your commits should be signed
 * Find an issue to work on or create a new issue. The issues are maintained at [openebs/openebs](https://github.com/openebs/openebs/issues). You can pick up from a list of [good-first-issues](https://github.com/openebs/openebs/labels/good%20first%20issue).
 * Claim your issue by commenting your intent to work on it to avoid duplication of efforts.
 * Fork the repository on GitHub.
-* Create a branch from where you want to base your work (usually master).
+* Create a branch from where you want to base your work (usually `main` or `develop`).
```
Comment (akhilerm) [inline CONTRIBUTING.md:16]: Why is develop added here?

**Review by akhilerm — CHANGES_REQUESTED** [review]
@kmova some changes are requested. PTAL.

File: .github/workflows/pull_request.yml (hunk 2 above)
Comment (kmova) [reply .github/workflows/pull_request.yml:21]: was planning on removing it later. but may be better to remove now. (done)

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Review by akhilerm — APPROVED** [review]
LGTM

**Review by mynktl — APPROVED** [review]

**Pull request merged**